Services/JWSProviderInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Services;
+
+/**
+ * Interface for JWS provider classes.
+ *
+ * @since 2.0
```

Comment:
It's up to you, but I don't see the `@since` very often - with git/GitHub, it's so easy to find this information, if you actually need it (and then you don't have to keep it up to date manually)